README.md

```diff
@@ -90,7 +90,11 @@ const renderforest = new Renderforest({ signKey: '<signKey>', clientId: -1 })
 
 const payload = {
   limit: 2,
-  offset: 10
+  offset: 10,
+  includeApiProjects: false,
+  order: 'DESC', //possible options: 'DESC', 'ASC'
+  orderBy: 'order', //possible options: 'order', 'date'
+  search: '' //get projects with specified title
```

Comment:
does it mean that search is optional? also, `order` and `orderBy` does not make a sense. 
For example, you can do not include `orderBy` property if you don't want to use orderBy: "date".
In that case, `orderBy` can be optional too.

Reply:
Also, you can provide an example for search property.